src/Symfony/Component/DependencyInjection/Attribute/AutowireServiceClosure.php

```diff
@@ -20,8 +20,12 @@
 #[\Attribute(\Attribute::TARGET_PARAMETER)]
 class AutowireServiceClosure extends Autowire
 {
-    public function __construct(string $service)
-    {
+    public function __construct(
+        /*
```

Comment:
Should we use one or two `*`? Should be the same everywhere

Reply:
Fabbot would like only one it seems, so I'd say one :+1:

Reply:
It looks like fabbot requires `/**` when CPP is used and `/*` otherwise

Reply:
Indeed, commenting just before the parameters only works for the CPP 😞. It allows access write documentation once and access on both the argument and the property. It also avoids having to repeat the name and type of the argument.

Reply:
this only works for CPP indeed, because only CPP defines a property there. And the phpdoc spec supports phpdoc on properties or on methods, but not inline on arguments.

Reply:
Alright, thank you! I updated the whole PR by using `/**` on CPP and `@param` in other case. You can see example of this in the different value resolver like `MapRequestPayload`.